### PR TITLE
166409937 Host app on heroku

### PR DESCRIPTION
**Pull request by misocho**
#### Title
Host app on Heroku

#### Description
As it is, the app is currently not hosted on Heroku. These PR hosts the app on Heroku

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue) 

- [ ] New feature (non-breaking change which adds functionality) 

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 

- [ ] This change requires a documentation update

- [x] Task/Chore
#### How Has This Been Tested?
- Follow the link to the deployed staging app [heroku staging link](https://wger-fortem-staging.herokuapp.com/)
- Follow the link to the deployed production app [heroku production link](https://wger-fortem.herokuapp.com/en/software/features) 
#### Acceptance Criteria: 
- [x] The app works on heroku

#### Checklist:
- [x] App hosted on heroku
- [x] Staging and production pipelines created
#### PT stories
#166409937

### CODE REVIEW

File: wger/settings.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import os
+# Use 'DEBUG = True' to get more details for server errors
+DEBUG = True
+TEMPLATES[0]['OPTIONS']['debug'] = True # noqa
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+ADMINS = (
+    ('Your name', 'your_email@example.com'),
+)
+MANAGERS = ADMINS
```
Comment (martinMutuma) [inline wger/settings.py:14]: Good work getting the app working on heroku. Please consider moving the ADMINS setting to env and also provide .env.sample for future setups.

**Review by martinMutuma — CHANGES_REQUESTED** [review]

File: wger/settings.py (hunk 1 above)
Comment (misocho) [reply wger/settings.py:14]: Changes made

**Review by justMuriithi — APPROVED** [review]
Great job getting the hosting up and running on Heroku.

**Review by martinMutuma — APPROVED** [review]
Good work , Providing .env.sample will make it easier to implement the setups locally

**Review by michael-basweti — APPROVED** [review]
This does what it is expected to do

**Pull request merged**